### DISCUSSION TITLE
Help with setup

### DISCUSSION THREAD

**Discussion by tobes49** [hultenvp/solis-sensor#122] [General]
Hello,

Im trying to set this integration up but seem to not like my setup!

I installed via HACS
when I try to add the line to the configration.ymal file I get an error
**can not read a block mapping entry; a multiline key may not be an implicit key at line 27, column 20:
        portal_username: "portal_username"
                       ^**

Can anyone help with this??

Thank you

**Comment by tobes49** [comment]
It would appear the copy and paste on my android phone caused the issue,

Just redid the setup on a desktop computer and it now happy with everything.

**Comment by azuberony** [comment]
![456](https://github.com/user-attachments/assets/0fabdcd9-543d-4ba7-a683-788cbae66364)


Hello, does anyone have a picture of the correctly connected sensors in the energy dashboard

**Comment by patduckless** [comment]
![image](https://github.com/user-attachments/assets/8e4639d5-ea99-4ca7-b585-48cb9f44d913)
